script.js

```diff
@@ -5,99 +5,122 @@ function Player(id, figure, name){
    this.name=name;
 }
 
-$(document).ready(function(){
-   var noWinners=true,
-       whoWon,
-       playerPers,
-       playerMach,
-       possibleFig={
-          'xis' : 'X',
-          'circle' : 'O'
-       };
+function UI(){
+   this.possibleFig={
+      'xis' : 'X',
+      'circle' : 'O'
+   };
+}
+
+UI.prototype.HideDialog = function(){
+   $("#overlay").hide();
+   $("#dialog").fadeOut();
+}
 
-   ShowDialog();
+UI.prototype.showDialog = function (){
+   $("#overlay").show();
+   $("#dialog").fadeIn();  
+}
 
+UI.prototype.startGame = function(){
+   var self = this;
    $('button').on("click", function(){
       var chosen =this.id;
+      var persFig = self.possibleFig[chosen];
       var idMachine =  (chosen == 'xis') ? 'circle' : 'xis';
-      playerPers = new Player(chosen,possibleFig[chosen], 'you');
-      playerMach = new Player (idMachine,possibleFig[idMachine], 'machine');
-      HideDialog();
-      playMachine();
+      var machFig = self.possibleFig[idMachine];
+      var playerPers = new Player(chosen,persFig, 'you');
+      var playerMach = new Player (idMachine,machFig, 'machine');
+      self.HideDialog();
+      var game = new Game(playerMach,playerPers);
```

Comment:
isso deveria ir no construtor pro game ficar acessivel em outras partes.

`this.game = new Game()`